backend/src/main/java/club/devcord/devmarkt/repositories/TemplateRepo.java

```diff
@@ -31,6 +33,12 @@ public interface TemplateRepo extends CrudRepository<Template, Integer> {
 
   boolean existsByName(String name);
 
+  @Override
+  @Where("true")
```

Comment:
This looks wrong, can you explain the reason behind this?

Reply:
The Entity has a @Where annotation, which generates a WHERE statement in the sql query. If we annotate a method in the repo with this annotation, we're overriding the Entity one's. Since Micronaut doesn't allow us to not include this WHERE in the query (or at least I didn't find the way to do this) we're just override it with an expression, that evaluates always to "true".